app/models/picture.rb

```diff
@@ -0,0 +1,13 @@
+class Picture < ActiveRecord::Base
+  has_attached_file :image,
+                    :styles => { :medium => "300x300>", :thumb => "100x100>" },
+                    :default_url => "/images/:style/missing.png"
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def build_comment(comment_params, current_user)
+    comment = comments.build({comment: comment_params[:comment], user: current_user})
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.